### PR TITLE
feat: Add option to init with a pyproject.toml

### LINKED ISSUE #1177: pixi init pyproject.toml
### Feature request

A pixi init option to create a pyproject.toml file rather than a pixi.toml

### PR DESCRIPTION
**Pull request by olivier-lacroix**
Fixes #1177

### CODE REVIEW

File: src/cli/init.rs [hunk 1]
```diff
@@ -69,6 +75,36 @@ default = { solve-group = "default" }
...
 {%- endfor %}
 
+[tool.pixi.tasks]
+
+"#;
+
+/// The pyproject.toml template
+///
+/// This is used to create a pyproject.toml from scratch
+const NEW_PYROJECT_TEMPLATE: &str = r#"[project]
```
Comment (olivier-lacroix) [inline src/cli/init.rs:85]: Not 100% sure or what to include in that template. build-system, python version, etc.

Let me know what you think.

**Comment by olivier-lacroix** [comment]
Slightly related question: the import command initialise the prefix. This is currently the only branch that does this.

To be consistent, should we do it here as well (to install python)? Personally, I find it strange, and would rather pull out the prefix initialisation from the import branch

File: src/cli/init.rs (hunk 1 above)
Comment (tdejager) [reply src/cli/init.rs:85]: I know uv has a slower code path for it currently, but would hatch be a better option than setuptools?

**Comment by tdejager** [comment]
Yes! I ❤️ this thank you! With regards to your question I would keep this as you have now.

File: src/cli/init.rs (hunk 1 above)
Comment (olivier-lacroix) [reply src/cli/init.rs:85]: Maybe. Setuptools seems to remain the most used today. https://framapiaf.org/@fcodvpt/111540079686191842

Comment (tdejager) [reply src/cli/init.rs:85]: Ok let's leave it.

**Review by ruben-arts — APPROVED** [review]
Thanks @olivier-lacroix

**Pull request merged**